thrift/src/main/thrift/atoms/interactive.thrift

```diff
@@ -11,4 +11,11 @@ struct InteractiveAtom {
   4: required string html
   5: optional string mainJS
   6: optional string docData
+
+  /*
+  When an interactive is within an amp-iframe,
+  a placeholder field is required to for the iframe to be displayed at the top of the AMP.
+  See here: https://amp.dev/documentation/components/amp-iframe/ for more information.
+   */
+  7: optional string placeholder
```

Comment:
Any opinions over `placeholder` vs `placeholderUrl` as the field name?

Reply:
if it is an url, probably better placeholderUrl I think... but also you could say it is for the amp-iframe in the name maybe? Or is it obvious already? ampIframeUrl? or ampIframePlaceholderUrl?

Reply:
Hmm - I was talking to @ajwl about including amp in the name. Probably splitting hairs, but I'm not sure we need to be that specific. If I understand correctly, it's a url for a placeholder image that will be used by amp, but there's nothing amp specific about it (?)

Reply:
sure, if it is not specific to this, fair enough!

Reply:
Agree that `placeholderUrl` is probably the best fit for this